src/ui/profile_preferences.ui

```diff
@@ -1341,124 +1341,90 @@
        <string>Mapper</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_24">
-       <item row="1" column="0">
-        <widget class="QGroupBox" name="downloadMapOptions">
+       <item row="4" column="0">
```

Comment:
I have a commit I would like to put forward to put into this series that does the reordering and makes the overall changes easier to spot - and as a result - I did find that `label_mapFileActionResult` has been squashed into one third of the width of the dialogue whereas it was the full width before.

I also would like to restore the, now emptied, `groupBox_debug` as it being present on the form means it is possible to simply drop in temporary run-time debugging controls for particular builds in a place set-aside for them. To reflect that there is not anything there I would leave it disabled (and thus greyed-out) by default, but it can obviously be re-enabled programmatically in a single code-line!